vivado.py

```diff
@@ -235,7 +235,7 @@ def __init__(self, rootdir):
         Vivado.__init__(self, rootdir)
         self.synthtool = 'yosys'
         self.synthoptions = ['-iopad', '-arch xc7', '-flatten']
-        self.toolchain = 'yosys-vivado'
+        self.toolchain = 'vivado-yosys'
```

Comment:
I think this was written as `yosys-vivado` as it better shows the fact that first yosys is run for synthesis and than vivado does P&R.